service/history/workflowTaskHandler.go

```diff
@@ -641,13 +641,11 @@ func (handler *workflowTaskHandlerImpl) handleCommandContinueAsNewWorkflow(
 		return handler.failCommand(enumspb.WORKFLOW_TASK_FAILED_CAUSE_UNHANDLED_COMMAND, nil)
 	}
 
-	executionInfo := handler.mutableState.GetExecutionInfo()
```

Comment:
this line should be kept so below can use `executionInfo` rather than `handler.mutableState.GetExecutionInfo()` multiple times?

Reply:
Initially I wanted to remove all these shortcuts, but then I realized how many of them we have and gave up. I think performance penalty is minimal but code readablity is better. You don't need to remember what is `executionInfo` and where it comes from. And this is pretty big func. Using var which is declared 100 lines above is not good.